tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,8 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # assert item not in vendor.inventory
+    assert result == False
```

Comment:
Great start! It would also be good to check the length of the inventory to make sure nothing was removed accidentally.